packages/multimedia/zvbi/package.mk

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2020-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="zvbi"
+PKG_VERSION="0.2.35"
+PKG_SHA256="fc883c34111a487c4a783f91b1b2bb5610d8d8e58dcba80c7ab31e67e4765318"
+PKG_LICENSE="GPL"
```

Comment:
```suggestion
PKG_LICENSE="GPL2+"
```

Reply:
at least the readme stats this :)

Reply:
Thanks, I may have got the licences the wrong way around - will confirm and fix!

Reply:
I think [zvbi](https://sourceforge.net/p/zapping/svn/HEAD/tree/tags/zvbi-0-2-35/vbi/COPYING) is actually GPL2 (albeit with an option to support later versions), and [inputstream.adaptive](https://github.com/xbmc/inputstream.ffmpegdirect/blob/72a12ac3a9aee9a6c2e54d1f704d27ef163de2b0/debian/copyright) is explicitly GPL2+ based on my reading of the included licences, so I'll change zvbi to GPL2.